client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -92,6 +94,7 @@ export class DomainDetailsForm extends PureComponent {
 
 		this.inputRefs = {};
 		this.inputRefCallbacks = {};
+		this.shouldFocusAddress = false;
```

Comment:
I think we should add a comment here mentioning that we'd prefer to use a callback ref on the top level form, but there are extra updates from the formstate that prevent that, and that we're using a property on the instance itself because this is state that we don't want to trigger state updates.